### PR TITLE
add exclude pattern

### PR DESCRIPTION
**Pull request by mrmasterplan**

### CODE REVIEW

File: tests/cluster/delta/test_sparkexecutor.py [hunk 1]
```diff
@@ -33,8 +33,10 @@ def tearDownClass(cls):
         cls.dh.from_tc("SparkTestTable1").drop()
 
     def test_can_execute(self):
-        SparkSqlExecutor().execute_sql_file("test1")
+        SparkSqlExecutor().execute_sql_file("*", exclude_pattern="debug")
 
         self.dh.from_tc("SparkTestTable1").read()
```
Comment (LauJohansson) [inline tests/cluster/delta/test_sparkexecutor.py:38]: To ensure that this actually tests if exclude works, the test should check that neither db2 or table2 exists here.

**Review by LauJohansson — CHANGES_REQUESTED** [review]

File: tests/cluster/delta/test_sparkexecutor.py (hunk 1 above)
Comment (LauJohansson) [reply tests/cluster/delta/test_sparkexecutor.py:38]: table1_name = self.tc.table_name("SqlTestTable1")
sql_argument = f"""
        (SELECT * FROM INFORMATION_SCHEMA.TABLES
        WHERE TABLE_NAME = '{table1_name}') target
        """
table_exists = self.sql_server.load_sql(sql_argument)
self.assertEqual(table_exists.count(), 0)

Comment (LauJohansson) [reply tests/cluster/delta/test_sparkexecutor.py:38]: The code is taken/inspired by the deliverysql-test in atc.

Comment (mrmasterplan) [reply tests/cluster/delta/test_sparkexecutor.py:39]: thanks for the suggestion. I added this line
```
self.assertEqual(Spark.get().sql("SHOW DATABASES LIKE my_db2*").count(), 0)
```

**Pull request merged**